lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -38,6 +39,7 @@ class RenderForm extends React.Component {
   submitForm() {
     const url = 'https://arbisoft.zendesk.com/api/v2/tickets.json',
       $userInfo = $('.user-info'),
+      $this = this,
       request = new XMLHttpRequest(),
       $course = $('#course'),
       accessToken = 'd6ed06821334b6584dd9607d04007c281007324ed07e087879c9c44835c684da',
```

Comment:
This looks like a real access token. This shouldn't be committed to the codebase. Same goes for the URL above. In fact, now that I see we are in edx/edx-platform, you should invalidate that token ASAP.

Reply:
@clintonb This was a trail account on Zendesk to test this change. Access token and URL will be configured through settings in https://openedx.atlassian.net/browse/LEARNER-2736

Reply:
because my trail account is expired, I am removing url and token for now.